internal/bminventory/inventory.go

```diff
@@ -133,7 +133,7 @@ type InstallerInternals interface {
 	UpdateHostInstallerArgsInternal(ctx context.Context, params installer.UpdateHostInstallerArgsParams) (*models.Host, error)
 	UpdateHostIgnitionInternal(ctx context.Context, params installer.UpdateHostIgnitionParams) (*models.Host, error)
 	GetCredentialsInternal(ctx context.Context, params installer.GetCredentialsParams) (*models.Credentials, error)
-	DownloadClusterKubeconfigInternal(ctx context.Context, params installer.DownloadClusterKubeconfigParams) (io.ReadCloser, int64, error)
+	DownloadClusterKubeconfigInternal(ctx context.Context, params installer.DownloadClusterKubeconfigParams, kubeconfig constants.KubeconfigType) (io.ReadCloser, int64, error)
```

Comment:
maybe instead of using `DownloadClusterKubeconfigInternal` it should be replaced with `DownloadClusterFilesInternal` that already have a filename parameter.

Reply:
Done